compiler/modules/types/core.bal

```diff
@@ -290,6 +291,7 @@ type BasicTypeOps readonly & record {|
     BinOp diff = binOpPanic;
     UnaryOp complement = unaryOpPanic;
     UnaryTypeCheckOp isEmpty = unaryTypeCheckOpPanic;
+    UnaryTypeCheckOp isFinite = unaryTypeCheckOpPanic;
```

Comment:
This doesn't feel right to me. Do we need isFinite for all basic types?

Reply:
fixed. (We only need this for list and mapping types)